src/interfaces/Config.ts

```diff
@@ -79,13 +79,20 @@ export interface IChartConfig extends IMeasuresStackConfig {
     secondaryChartType?: VisualizationObject.VisualizationType;
 }
 
+export interface IStackMeasuresConfig {
+    stackMeasuresToPercent?: boolean;
+    series?: ISeriesItem[];
+    yAxis?: any[];
+}
+
 export interface IAxisConfig {
     visible?: boolean;
     labelsEnabled?: boolean;
     rotation?: string;
     min?: string;
     max?: string;
     measures?: string[];
+    stackLabels?: boolean;
```

Comment:
I don't see your new code lines refering to `stackLabels`.
Should we need it?

Reply:
Removed. Its my mistake :)